osu.Game.Rulesets.Osu/UI/Cursor/GameplayCursor.cs

```diff
@@ -23,17 +23,25 @@ public class GameplayCursor : CursorContainer, IKeyBindingHandler<OsuAction>
 
         protected override Container<Drawable> Content => fadeContainer;
 
+        private bool cursorExpand;
+
         private readonly Container<Drawable> fadeContainer;
 
+        private SkinnableCursor skinnableCursor;
+
         public GameplayCursor()
         {
-            InternalChild = fadeContainer = new Container
+            InternalChildren = new Drawable[]
             {
-                RelativeSizeAxes = Axes.Both,
-                Children = new Drawable[]
+                fadeContainer = new Container
                 {
-                    new CursorTrail { Depth = 1 }
-                }
+                    RelativeSizeAxes = Axes.Both,
+                    Children = new Drawable[]
+                    {
+                        new CursorTrail { Depth = 1 }
+                    }
+                },
+                skinnableCursor = new SkinnableCursor()
```

Comment:
You'll probably want this to be inside `FadeContainer` (before `CursorTrail`) and set it to the `Content =>` target.